docs/api.html

```diff
@@ -74,7 +74,7 @@
         <a style ="color:orangered" href="#events"> Events</a> &nbsp; |  &nbsp;
         <a style ="color:orangered" href="#slots"> Slots</a> &nbsp; |  &nbsp;
         <a style ="color:orangered" href="./css-parts.html"> Advance Styling </a> &nbsp; |  &nbsp;
```

Comment:
You might want to fix this one as well :)

```suggestion
        <a style ="color:orangered" href="./css-parts.html"> Advanced Styling </a> &nbsp; |  &nbsp;
```

Reply:
The same typo is present here as well, btw:

https://github.com/mrin9/RapiDoc/blob/2ab68d23eb66b7d04605498302546f38cdb0be57/docs/list.html#L185